libnetwork/drivers/bridge/setup_device_linux.go

```diff
@@ -48,6 +50,14 @@ func setupDevice(config *networkConfiguration, i *bridgeInterface) error {
 func setupMTU(config *networkConfiguration, i *bridgeInterface) error {
 	if err := i.nlh.LinkSetMTU(i.Link, config.Mtu); err != nil {
 		log.G(context.TODO()).WithError(err).Errorf("Failed to set bridge MTU %s via netlink", config.BridgeName)
+		// Before Linux v4.17, bridges couldn't be configured "manually" with an MTU greater than 1500, although it
+		// could be autoconfigured with such a value when interfaces were added to the bridge. In that case, the
+		// bridge MTU would be set automatically by the kernel to the lowest MTU of all interfaces attached. To keep
+		// compatibility with older kernels, we need to discard -EINVAL.
+		// TODO(aker): remove this once we drop support for CentOS/RHEL 7.
+		if errors.Is(err, syscall.EINVAL) {
```

Comment:
[The max MTU is capped to `ETH_MAX_MTU` on recent kernels](https://elixir.bootlin.com/linux/v6.6.15/source/net/bridge/br_device.c#L532) so we could be a bit more selective in when to ignore EINVAL.
```suggestion
		if config.Mtu > 1500 && config.Mtu <= 0xFFFF && errors.Is(err, syscall.EINVAL) {
```